src/binding.cc

```diff
@@ -610,6 +619,33 @@ void v8__PropertyCallbackInfo__GetReturnValue(
   *out = self.GetReturnValue();
 }
 
+void v8__SnapshotCreator__CONSTRUCT(uninit_t<v8::SnapshotCreator>& buf,
+                                    const intptr_t* external_references) {
+  construct_in_place<v8::SnapshotCreator>(buf, external_references);
+}
+
+void v8__SnapshotCreator__DESTRUCT(v8::SnapshotCreator& self) {
+  self.~SnapshotCreator();
+}
+
+v8::Isolate* v8__SnapshotCreator__GetIsolate(v8::SnapshotCreator& self) {
+  return self.GetIsolate();
+}
+
+void v8__SnapshotCreator__SetDefaultContext(v8::SnapshotCreator& self,
+                                            v8::Local<v8::Context> context) {
+  self.SetDefaultContext(context);
+}
+
+v8::StartupData v8__SnapshotCreator__CreateBlob(
+    v8::SnapshotCreator* self,
+    v8::SnapshotCreator::FunctionCodeHandling function_code_handling) {
+  printf("working fine here\n");
+  auto a = self->CreateBlob(function_code_handling);
+  printf("working fine here too\n");
+  return a;
+}
+
```

Comment:
@ry it crashes CI in this binding on 
```
auto a = self->CreateBlob(function_code_handling);
```
It works for me in debug build though; I wonder if it might have something to do with `SerializeInternalFieldsCallback` 🤔